romancal/jump/jump_step.py

```diff
@@ -38,22 +39,23 @@ class JumpStep(RomanStep):
     def process(self, input):
 
         # Open input as a Roman DataModel (single integration; 3D arrays)
-        with rdm.RampModel(input) as input_model:
+        with rdd.RampModel(input) as input_model:
 
             # Extract the needed info from the Roman Data Model
             meta = input_model.meta
             r_data = input_model.data
             r_gdq = input_model.groupdq
             r_pdq = input_model.pixeldq
             r_err = input_model.err
+            result = input_model.copy()
 
             frames_per_group = meta.exposure.nframes
 
             # Modify the arrays for input into the 'common' jump (4D)
-            data = np.broadcast_to(r_data, (1,) + r_data.shape)
-            gdq = np.broadcast_to(r_gdq, (1,) + r_gdq.shape)
-            pdq = np.broadcast_to(r_pdq, (1,) + r_pdq.shape)
-            err = np.broadcast_to(r_err, (1,) + r_err.shape)
+            data = r_data[np.newaxis, :].astype(np.float32)
```

Comment:
Are the astype castings needed if the input is a RampModel?

Reply:
Hmm... I don't think so. I'll see if Dave G. has anything to add before I remove them and see if he is guarding against a known issue.

Reply:
No - since the input is a RampModel the astypes are not needed.